src/frontend/pages/feilsider/ServerFeil.tsx

```diff
@@ -0,0 +1,41 @@
+import { BodyShort, Box, HGrid, Heading, Link, Page, VStack } from '@navikt/ds-react';
+import * as React from 'react';
+
+interface IProps {
```

Comment:
er I-prefixen litt unødvendig når vi ser enkelt her at det er et interface?
```suggestion
interface Props {
```